mjk-examples/ZCL/odmdata-zcl-transitiontimedata.sdf.json

```diff
@@ -10,13 +10,13 @@
   }, 
   "defaultNamespace": "zcl", 
   "odmData": {
-    "transitiontimedata": {
+    "transitionTimeData": {
       "type": "number", 
       "widthInBits": 16, 
       "minimum": 0, 
       "maximum": 65535, 
       "multipleOf": 1, 
-      "unit": "seconds", 
+      "scaleUnit": "seconds", 
```

Comment:
scaleUnits seems underspecified now. We should clarify what it's really for. Sand should use same SenML units? This should be "s"?

Reply:
It's now "unit": "s"